backend/src/src-moderation/src/main/java/org/eclipse/sw360/moderation/db/ModerationDatabaseHandler.java

```diff
@@ -207,52 +207,63 @@ public RequestStatus updateClearingRequest(ClearingRequest request, User user, S
                 request.unsetPriority();
             }
             ClearingRequest currentRequest = getClearingRequestByIdForEdit(request.getId(), user);
-            StringBuilder commentText = new StringBuilder("Clearing Request is updated: ");
-            if (!currentRequest.getClearingState().equals(request.getClearingState())) {
-                if (ClearingRequestState.CLOSED.equals(currentRequest.getClearingState())
-                        || ClearingRequestState.REJECTED.equals(currentRequest.getClearingState())) {
-                    commentText = new StringBuilder("Clearing Request is re-opened: ");
+
+            if (!request.equals(currentRequest)) {
```

Comment:
remove this condition from here and add it in `updateClearingRequest` function in `ModerationPortletUtils` class.